samples/nrf9160/modem_shell/src/main.c

```diff
@@ -147,10 +147,7 @@ void main(void)
 #if defined(CONFIG_MOSH_WORKER_THREADS)
 	th_ctrl_init();
 #endif
-#if defined(CONFIG_MOSH_GNSS_ENABLE_LNA)
-	gnss_set_lna_enabled(true);
-#endif
-
+	gnss_configure_lna();
```

Comment:
This needs to be flagged, otherwise compilation will fail when CONFIG_MOSH_GNSS=n.
```suggestion
#if defined(CONFIG_MOSH_GNSS)
	gnss_configure_lna();
#endif
```

Reply:
Try compilation with CONFIG_MOSH_GNSS=n to make sure it works.